projects/project-player-library/src/lib/constants/actionConstants.ts

```diff
@@ -92,5 +92,17 @@ export const actions = {
             action:"delete",
             color: "primary-icon"
         },
-    ]
+    ],
+    PROJECT_DOWNLOAD : {
```

Comment:
Why are we creating it as a new key
Cant we add this value in the PROJECT_ACTIONS array itself?

Reply:
made the changes